install_build_dependencies.sh

```diff
@@ -294,6 +294,44 @@ elif [ -f /etc/os-release ] && grep -q "alpine" /etc/os-release; then
         enchant2 \
         `# samples` \
         nlohmann-json
+elif [ -f /etc/os-release ] && \
+   (grep -q "tencentos" /etc/os-release || grep -q "opencloudos" /etc/os-release); then
```

Comment:
can it be merged with `elif [ -f /etc/redhat-release ] || grep -q "rhel" /etc/os-release ; then` branch?
Looks like both OSes use the same package manager

Reply:
> can it be merged with `elif [ -f /etc/redhat-release ] || grep -q "rhel" /etc/os-release ; then` branch? Looks like both OSes use the same package manager

done